source/SH3/arc/vfile.cpp

```diff
@@ -100,3 +102,22 @@ std::size_t sh3_arc_vfile::ReadData(void* destination, std::size_t len, read_err
 
     return nbytes;
 }
+
+void sh3_arc_vfile::Dump2Disk()
+{
+    if(!open || buffer.empty())
+    {
+        Log(LogLevel::WARN, "sh3_arc_vfile::Dump2Disk( ): Warning! Attempting to flush unopen or empty buffer to disk!");
+        return;
+    }
+
+    // Firstly, we need to convert the file path to just the filename.
+    std::stringstream ss(fname);
+    std::size_t pos = fname.rfind('/');
+
+    std::ofstream out_file(fname.substr(pos + 1, fname.size()));
+    if(!out_file)
+        return;
+
+    out_file.write((const char*)&buffer[0], buffer.size());
```

Comment:
Nah it doesn't, I get an `invalid conversion from unsigned char* to const char*` erroir